mgmt/RecordsConfig.cc

```diff
@@ -815,33 +815,33 @@ static const RecordElement RecordsConfig[] =
 #endif
    RECU_DYNAMIC, RR_NULL, RECC_INT, "[0-65535]", RECA_NULL}
   ,
-  {RECT_CONFIG, "proxy.config.net.sock_recv_buffer_size_in", RECD_INT, "0", RECU_NULL, RR_NULL, RECC_NULL, NULL, RECA_NULL}
+  {RECT_CONFIG, "proxy.config.net.sock_recv_buffer_size_in", RECD_INT, "0", RECU_RESTART_TS, RR_NULL, RECC_STR, "^[0-9]+$", RECA_NULL}
```

Comment:
If you give this a value of e.g. "234324sdf", it gives an error with the validator but without it just assigns 234324 to it. So it's better to have the validator, I think.